src/jdk.jdwp.agent/unix/native/libjdwp/exec_md.c

```diff
@@ -44,6 +48,104 @@ static char *skipNonWhitespace(char *p) {
     return p;
 }
 
+int
+isAsciiDigit(char c)
```

Comment:
The standard `isdigit` function from `ctype.h` does this.

Reply:
Done.